lib/elixir_newbie_web/components/responsive_layout.ex

```diff
@@ -0,0 +1,29 @@
+defmodule ElixirNewbieWeb.Components.ResponsiveLayout do
+  use ElixirNewbieWeb, :html
+
+  attr(:class, :string)
+  attr(:scroll_id, :string)
+  attr(:spacing, :string, values: ["full", "narrow", "none"], default: "narrow")
+  attr(:gap, :string, values: ["none", "tiny", "small", "large"], default: "small")
+  attr(:cols, :integer, values: [2, 3, 4])
+  slot(:inner_block, required: true)
+
+  def responsive_layout(assigns) do
```

Comment:
Interesting component! It looks like you're using it for both blogs and tags? Can you help me understand the purpose of it?

Reply:
This component is an abstraction of a section that have N elements that need be divided in columns. 
The main reason is just to avoid duplicated code (css classes) that can hide the real intention of the blog_list liveview.
The component also have a `data-scroll-id ` to be used with the scroll hook.